test/instances/router.lua

```diff
@@ -13,6 +13,7 @@ _G.vshard = {
     router = require('vshard.router'),
 }
 _G.ivshard = _G.vshard
+_G.ifiber = require('fiber')
 
```

Comment:
In the commit message, please, make a docbot request which describes the user visible changes. The template can be found here https://github.com/tarantool/docbot. Examples - in `git log`.

Reply:
Done.  As for the message, sorry for plagiarism but router's enabling/disabling logic is just the same as storage's one